source/api_reference/single_payment_reference/index.html.md.erb

```diff
@@ -152,13 +152,14 @@ After you send your request to get information about a payment, you’ll receive
 | `delayed_capture`                                 | boolean         | `delayed_capture` is `true` if you're controlling how long it takes GOV.UK Pay to take ('capture') a payment.<br><br>You can [read more about delaying taking a payment](/delayed_capture).                                                                                                                                                 |
 | `moto`                                             | boolean         | Indicates if this payment is a Mail Order / Telephone Order (MOTO) payment.<br><br>You can [read more about taking payments over the phone](/moto_payments).                                                                                                                                                                                     |
+| `corporate_card_surcharge`                         | number         | The corporate card surcharge amount in pence.<br><br>You can [read more about setting up corporate card fees](/corporate_card_surcharges/#add-corporate-card-fees). |
 | `total_amount`                                    | number          | Amount your user paid in pence, including corporate card fees.<br><br>`total_amount` only appears if you added a corporate card surcharge to the payment.<br><br>You can [read more about corporate card surcharges](/corporate_card_surcharges/#add-corporate-card-fees).                                                                   |
-| `fee`                                              | number          | The payment service provider's transaction fee, in pence.<br><br>`fee` only appears when we have taken ('captured') the payment from the user or if their payment fails after they submitted their card details.<br><br>You can [read more about PSP fees](/reporting/#psp-fees)                                                                |
-| `net_amount`                                      | number          | The amount, in pence, that'll be paid into your bank account after your payment service provider takes the `fee`.                                                                                                                                                                                                                                 |
+| `fee`                                              | number          | The payment service provider's transaction fee, in pence.<br><br>`fee` only appears when we have taken ('captured') the payment from the user or if their payment fails after they submitted their card details.<br><br>`fee` will not appear if your PSP is Worldpay or you are using an API key from a test service.<br><br>You can [read more about PSP fees](/reporting/#psp-fees)                                                                |
```

Comment:
have we already spelled out PSP on this page?

Reply:
We have, but I think the way user's will read the page is by jumping around. We say 'payment service provider' earlier in this line but don't then have PSP in brackets. I'll add this.